CIP-1694/README.md

```diff
@@ -898,24 +887,22 @@ Regardless of any governance mechanism, SPO participation is needed for any hard
 For this reason, we make their cooperation explicit in the hard fork initiation governance action,
 by always requiring their vote.
 The constitutional committee also votes, signaling the constitutionality of a hard fork.
-The DReps also vote, to represent the will of every stake holder
+The DReps also vote, to represent the will of every stake holder.
 
 ### New Metadata structures
 
 Both the governance actions and the votes use new metadata fields,
 in the form of URLs and integrity hashes
 (mirroring the metadata structure for stake pool registration).
 The metadata is used to provide context.
-For example, votes by the constitutional committee need an explanation as to why they are in line with
-the constitution.
```

Comment:
why did you remove this?

Reply:
Votes currently don't have anchors. So we'd have to add them if we wanted this.

Reply:
:100: